lib/libyear_bundler/cli.rb

```diff
@@ -56,11 +56,11 @@ def load_gemfile_path
     end
 
     def query
-      Query.new(@gemfile_path, @argv).execute
+      Query.new(@gemfile_path).execute
     end
 
     def report
-      @_report ||= Report.new(query)
+      @_report ||= Report.new(query, @argv)
```

Comment:
Hmm seems like this is going to conflict with https://github.com/jaredbeck/libyear-bundler/pull/8